src/style.css

```diff
@@ -1 +1,82 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 96px;
```

Comment:
Don't fix container size (if there is no such a requirement). The width is set to a fixed size which may not be flexible with the content. If there's no specific requirement for this fixed size, consider using a more responsive approach.